### ISSUE TITLE
Does windows_crashes only fetch data from the minidump folder and we can't extract any data unless we manually create a dump file

### ISSUE THREAD

**Issue by Vedang-Agarwal** [osquery/osquery#8270]
# Question

The question is regarding windows_crashes table on windows.
Currently I'm using windows 10 and osquery version 5.9.1 . 
Is there any way to fetch data in the osquery tables without having to manually create dump files everytime we need to see the osquery outputs?
Does osquery only fetch data from the minidump folder only?

**Comment by michael-myers** [comment]
Yes, `windows_crashes` exclusively reads `.dmp` (minidump) files from disk. It does not hook into the crash reporting pipeline in Windows. If you configure WER to write mini dumps automatically, it will populate the CrashDumps folder whenever there is a crash, and there shouldn't be any manual intervention necessary every time.

It is purely a retrospective view of already-written dump files; it is not an "evented" table and does not catch and observe crashes as they happen. If someone would like to contribute such a table they should open a blueprint first but it does sound like an interesting project.
